index.html

```diff
@@ -162,17 +167,40 @@ <h2>About Myself</h2>
     </section>
 
     <!-- Contact Section -->
-    <section class="contact-section">
+    <section class="contact-section" id="Contact">
       <div class="contact-container">
         <h2>Contact me</h2>
         <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
           I’d love to help with it!</p>
         <form action="https://formspree.io/f/mlekypop" method="POST">
-          <input type="text" class="input-container" placeholder="Full name" maxlength="30" name="fullname" id="fullname" required>
+          <input 
+            type="text" 
+            class="input-container" 
+            placeholder="Full name" 
+            maxlength="30" 
+            name="fullname" 
+            id="fullname" 
+            required
+          >
           <label class="labels" for="fullname">Full name</label>
-          <input type="email" class="input-container" placeholder="Email address" name="email" id="email" required>
+          <input 
+            type="email" 
+            class="input-container" 
+            placeholder="Email address" 
+            name="email" 
+            id="email" 
+            required
+          >
           <label class="labels" for="email">Email address</label>
-          <textarea type="text" class="input-container-2" placeholder="Write your message here..." maxlength="500" name="comment" id="comment" required></textarea>
+          <textarea 
+            type="text" 
+            class="input-container-2" 
+            placeholder="Write your message here..." 
+            maxlength="500" 
+            name="comment" 
+            id="comment" 
+            required
+          ></textarea>
           <label class="labels" for="comment">Comment</label>
           <button type="submit" class="button-form">Get in touch</button>
```

Comment:
Greeting!
You did an amazing job, but there is a little thing you'll need to work on to go through the next steps.

According to the figma design, I suggest you center this button to make your design match the Desktop version in the Figma design.

> The Figma Dsign
>
><img width="780" alt="Screenshot 2023-06-08 at 5 02 23 PM" src="https://github.com/MasterLeiva/DavidLeiva-Portfolio/assets/91237525/cb05c40b-fcde-4c8f-bc28-7662795d57d3">
>
> ## Your Design
>
><img width="1081" alt="Screenshot 2023-06-08 at 5 02 56 PM" src="https://github.com/MasterLeiva/DavidLeiva-Portfolio/assets/91237525/fe152097-ecda-49e7-90c4-868a7c8d4171">